btree/btree.c

```diff
@@ -3746,11 +3791,12 @@ int  btree_sibling_first_key_get(struct m0_btree_oimpl *oi, struct td *tree,
 				node_child(s, &child);
 				if (!address_in_segment(child))
 					return M0_ERR(-EFAULT);
+				i++;
 				node_get(&oi->i_nop, tree,
 					 &child, P_CLEANUP);
 				s->s_idx = 0;
 				s->s_node = oi->i_nop.no_node;
-				i++;
+				oi->i_level[i].l_sibling = oi->i_nop.no_node;
```

Comment:
Please refer the same comment below .

Reply:
I am planning to modify the get_tick SLANT code also, to work in unlock mode. For that I will need sibling node. Just to have coherent code, I am storing l_sibling and clearing up it in level_cleanup(). We anyways had allocated memory for l_sibling, so assigning a node to this will not cause any performance degradation.

Reply:
okay